tests/single/bundle/test_paymaster.py

```diff
@@ -0,0 +1,60 @@
+import collections
+import pytest
+
+from eth_utils import to_hex
+
+from tests.types import UserOperation, RPCErrorCode
+from tests.utils import (
+    assert_ok,
+    assert_rpc_error,
+    deploy_wallet_contract,
+    deploy_contract,
+)
+
+
+def to_number(x):
+    return x if isinstance(x, (int, float)) else int(x, 16)
+
+
+def sum_hex(*args):
+    return sum(to_number(i) for i in args if i is not None)
+
+
+def get_userop_verification_max_cost(userOp):
+    return sum_hex(userOp.preVerificationGas,
+                   userOp.verificationGasLimit,
+                   userOp.paymasterVerificationGasLimit) * to_number(userOp.maxFeePerGas)
```

Comment:
Do we have a 'utils.py'? Let's have a 'utils.py'. And let's move these functions to 'utils.py'?